src/io/flutter/preview/RenderHelper.java

```diff
@@ -146,6 +163,8 @@ private static void render(@NotNull RenderRequest request) {
       template = template.replace("// TODO: import library to render", "import '" + toRenderFile.toURI() + "';");
       template = template.replace("new Container(); // TODO: create widget", widgetCreation);
       template = template.replace("{}; // TODO: use flutterDesignerWidgets", "flutterDesignerWidgets;");
+      template = template.replace("350.0 /*TODO: width*/", request.width + ".0");
```

Comment:
I would recommend having the template have an entry something like
__WINDOW_WIDTH_UID1239394032__
instead of matching on something that doesn't look like a template hole as far as a developer unfamiliar with the code is concerned.
so that filling out the holes in the template is more robust and less surprising.